app/uk/gov/hmrc/individualsifapistub/domain/TaxCredits.scala

```diff
@@ -78,11 +78,13 @@ object TaxCredits {
   def isMultipleOfOne(value: Double): Boolean = value % 1 == 0
   def isMultipleOfPointZeroOne(value: Double): Boolean = (value * 100.0) % 1 == 0
```

Comment:
This needs updating to cast value to BigDecimal as this was a bug fix for some numbers failing this test. Check Income.scala : Line 146 to see example.